.github/workflows/conventional-commits.yml

```diff
@@ -11,7 +11,6 @@ on:
 
 permissions:
   contents: read
-  pull-requests: read
```

Comment:
JIC anyone else comes across this -- the `pull-requests: read` permission is only required if `update-labels` is true. We configured it to be `false`.

See: https://github.com/dev-build-deploy/commit-me/blob/main/docs/github-action.md#pull-request-and-associated-commit-scanning

Reply:
ah, since this is a public repo, in general everyone has read permissions to pull-requests and so you don't need to specify it (even if you did need it)

Reply:
Ah, TIL! Thank you!